### DISCUSSION TITLE
Better mod selection UI

### DISCUSSION THREAD

**Discussion by Kyorie** [ppy/osu#15344] [Ideas] [answered]
1. I would like to see BPM/AR/OD etc. changes as i change something in mod customisation menu. For now, i can't see it because mod selection takes too much space.
2. Also i think that mod customisation menu should not overlap mods on selection menu
3. Seems like there is a 1px gap

![image](https://user-images.githubusercontent.com/47324490/139231641-601bdd02-2ddb-4992-8559-f6e378485211.png)

**Comment by peppy** [comment, marked as answer]
a complete update to this ui is already completed and just need implementation.
